templates/org/settings/actions.tmpl

```diff
@@ -0,0 +1,9 @@
+{{template "org/settings/layout_head" (dict "ctxData" . "pageClass" "organization settings actions")}}
+	<div class="org-setting-content">
+	{{if eq .PageType "runners"}}
+		{{template "shared/actions/runner_list" .}}
+	{{else if eq .PageType "secrets"}}
+		{{template "shared/secrets/add_list" .}}
+	{{end}}
```

Comment:
Why we need thses `if` check, I guess the templates should be called clearly by backedn?

Reply:
I will move these to backend, but there might be if statements in frontend for different layout sub template:

<img width="1040" alt="Screen Shot 2023-04-25 at 11 27 27" src="https://user-images.githubusercontent.com/17645053/234167582-db677d97-54c9-4f6a-a7b6-cb3d889f79f0.png">

I does not find a way to pass setting type as a variable to `{{template }}`..

Reply:
Maybe we need to wait for layout.

Then keep it as-is at the moment.